python/turbine_models/custom_models/stateless_llama.py

```diff
@@ -227,6 +234,8 @@ def forward(token0: torch.Tensor, *state0_flat):
         )
         with open(f"{safe_name}.vmfb", "wb+") as f:
             f.write(flatbuffer_blob)
+        print("saved to ", safe_name + ".vmfb")
+        exit()
```

Comment:
Extra debugging to remove?

Reply:
This is more of a quick and dirty fix to an error not returning anything back to the main function when compiling to vmfb. We are done at this point so I'm just exiting, figured true cleanup would wait for the refactor